src/OptionSettlementEngine.sol

```diff
@@ -389,6 +396,7 @@ contract OptionSettlementEngine is ERC1155, IOptionSettlementEngine {
             nextClaimKey: 1
         });
 
+        // Emit event.
```

Comment:
Stuff like this is unneeded, let's use comments sparingly where they add clarity on complex operations. It's obvious to the reader that an event is being emitted here.

Reply:
It kinda breaks the pattern we establish, and I was going back and forth. But okay, yes. I do like those clarifier comments I added above require statements FWIW — but that does help readability